pages/test_analytics/test_executions.md

```diff
@@ -0,0 +1,16 @@
+# Test executions
+
+Each [Buildkite plan](https://buildkite.com/pricing) has test execution inclusions, which vary depending on the plan type and the number of users in your organization.
+
+You can find the test execution count on each run page
```

Comment:
The sentence needs to end with either a period or colon. To make it more parallel with the Pipelines page, perhaps:

```suggestion
You can find the test execution details for a run at the bottom of the run page, and your organization's [total usage](#usage-page) in Settings.
```